drivers/all_neighbors_system_dashboard/app/models/all_neighbors_system_dashboard/enrollment_attribute_calculations.rb

```diff
@@ -116,66 +116,48 @@ def race_list(enrollment)
       end
 
       def ce_infos_for_batch(filter, batch)
```

Comment:
Where is this being called?

Reply:
In `Report#populate_universe` -- this is a concern not because it gets used more than once, but just to decompose the different aspects of the report calculations.